clang/lib/Basic/Targets/X86.h

```diff
@@ -819,6 +819,43 @@ class LLVM_LIBRARY_VISIBILITY X86_64TargetInfo : public X86TargetInfo {
   }
 };
 
+// x86-64 UEFI target
+class LLVM_LIBRARY_VISIBILITY UEFIX86_64TargetInfo
+    : public UEFITargetInfo<X86_64TargetInfo> {
+public:
+  UEFIX86_64TargetInfo(const llvm::Triple &Triple, const TargetOptions &Opts)
+      : UEFITargetInfo<X86_64TargetInfo>(Triple, Opts) {
```

Comment:
This clangBasic change needs a `clang/test/CodeGen` test for things including: data layout , calling convention.

Reply:
I have added layout test to target-data.c. I am trying to figure out where to add a test for the calling convention.

Reply:
Still waiting for a calling convention test

Reply:
Please see `clang/Driver/unittests/ToolchainTest.cpp`

https://github.com/llvm/llvm-project/pull/76838/files#diff-2a39548e3b4a62a12affcb6dbe9e8f563cddc2aa50aa7e24b94a1de0bb91be1eR578